### PR TITLE
analytics overhaul

### PR DESCRIPTION
**Pull request by noslouch**
to comply with the new generalized spec

relies upon:
nypublicradio/publisher#452
nypublicradio/nypr-account-settings#54
nypublicradio/nypr-audio-services#9
nypublicradio/nypr-django-for-ember#8
nypublicradio/nypr-metrics#16
nypublicradio/nypr-player#8
nypublicradio/nypr-publisher-lib#37
nypublicradio/nypr-ui#90

### CODE REVIEW

File: app/components/discover-show/component.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ export default Component.extend({
   classNames:['discover-show'],
 
   classNameBindings:['isSelected'],
-  attributeBindings:['slug:data-slug'],
+  attributeBindings:['slug:data-slug', 'data-category', 'data-action', 'data-label', 'data-label'],
```
Comment (mike-hearn) [inline app/components/discover-show/component.js]: Do you want two `'data-label'`s here?

File: app/components/discover-topic-list/template.hbs [hunk 2]
```diff
@@ -4,7 +4,14 @@
...
   tagName='div'
   onchange=(action 'onMultiselectChangeEvent') as |topic isSelected|}}
-  {{discover-topic title=topic.title url=topic.url isSelected=isSelected}}
+  {{discover-topic
+    title=topic.title
+    url=topic.url
+    isSelected=isSelected
+    class='gtm__click-tracking'
+    data-action='Discover'
+    data-label=(concat (if isSelected 'Deselected Topic ' 'Selected Topic ') topic.title)
```
Comment (mike-hearn) [inline app/components/discover-topic-list/template.hbs]: I might be misunderstanding something here, but if `isSelected` is true, wouldn't this be a Selected Topic?

Comment (mike-hearn) [reply app/components/discover-topic-list/template.hbs]: Ah, nope, just figured it out, this is a click action which means if it's selected, clicking deselects it. Got it. Moving on.

File: app/components/discover-show/component.js (hunk 1 above)
Comment (noslouch) [reply app/components/discover-show/component.js]: nice cathc. I'll remove

File: app/components/discover-topic/component.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ import Component from '@ember/component';
 
 export default Component.extend({
   classNameBindings:['isSelected'],
-  classNames: ['discover-topic discover-topic-bubble'],
+  classNames: ['discover-topic', 'discover-topic-bubble'],
+  attributeBindings: ['data-category', 'data-action', 'data-label', 'data-label'],
```
Comment (mike-hearn) [inline app/components/discover-topic/component.js]: The double `data-label` is also here, fyi

**Pull request merged**